packaging/rpm/microshift.spec

```diff
@@ -265,55 +265,89 @@ systemctl enable --now --quiet openvswitch || true
 %{_bindir}/configure-ovs.sh
 %{_bindir}/configure-ovs-microshift.sh
 
+# Use Git command to generate the log and replace the VERSION string
+# LANG=C git log --date="format:%a %b %d %Y" --pretty="tformat:* %cd %an <%ae> VERSION%n- %s%n" microshift.spec
```

Comment:
Does this limit the list of changes to commits that touch this file? Is that what we want?

Reply:
Looking at the procedure Step 6 of [this document](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/rpm_packaging_guide/index#modifying-a-spec-file_working-with-spec-files), it says.

> The last section, %changelog, is a list of datestamped entries for each Version-Release of the package. They log packaging changes, not software changes.

MicroShift change log is more of the Release Notes, isn't it?

Reply:
Hmm, according to that you're right @ggiguash , I always got it wrong then '':-D

Reply:
Ah, that makes sense.